adapter/pkg/operator/utils/utils.go

```diff
@@ -119,6 +120,23 @@ func TieBreaker[T metav1.Object](k8sObjects []T) *T {
 	return &selectedk8sObject
 }
 
+// SelectPolicy selects the policy based on the policy override and default values
```

Comment:
Shouldn't the order be like:
```
	if !reflect.ValueOf(*policyUpOverride).IsZero() {
		return policyUpOverride
	}
	if !reflect.ValueOf(*policyDownOverride).IsZero() {
		return policyDownOverride
	}
	if !reflect.ValueOf(*policyDownDefault).IsZero() {
		return policyDownDefault
	}
	if !reflect.ValueOf(*policyUpDefault).IsZero() {
		return policyUpDefault
	}
```
But still we are not combining the values from two policies. According to the [policy spec](https://gateway-api.sigs.k8s.io/references/policy-attachment/), ideally user should be able to define `unit` in `default` section of api level RL policy and  include different `requestPerUnit` in `override` section of of resource level RL policy. It can be addressed in https://github.com/wso2/apk/issues/691 issue later as well